clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/arrays/TransferArraysV1Metadata.scala

```diff
@@ -0,0 +1,151 @@
+package org.broadinstitute.clio.transfer.model.arrays
+
+import java.net.URI
+import java.time.OffsetDateTime
+
+import org.broadinstitute.clio.transfer.model.TransferMetadata
+import org.broadinstitute.clio.util.model.DocumentStatus
+
+case class TransferArraysV1Metadata(
+  autocallCallRate: Option[Double] = None,
+  autocallDate: Option[OffsetDateTime] = None,
+  autocallGender: Option[Symbol] = None,
+  autocallPf: Option[Boolean] = None,
+  autocallVersion: Option[Int] = None,
+  beadPoolManifestFile: Option[URI] = None,
+  callRate: Option[Double] = None,
+  callRateThreshold: Option[Float] = None,
+  chipType: Option[String] = None,
+  clusterFile: Option[URI] = None,
+  controlSampleName: Option[String] = None,
+  createdAt: Option[OffsetDateTime] = None,
+  dbSnpVcf: Option[URI] = None,
+  dbSnpVcfIndex: Option[URI] = None,
+  documentStatus: Option[DocumentStatus] = None,
+  extendedChipManifestFile: Option[URI] = None,
+  filteredSnps: Option[Long] = None,
+  fingerprintPath: Option[URI] = None,
+  fingerprintingDetailMetricsPath: Option[URI] = None,
+  fingerprintingSummaryMetricsPath: Option[URI] = None,
+  fpGender: Option[Symbol] = None,
+  genderClusterFile: Option[URI] = None,
+  genderConcordancePf: Option[Boolean] = None,
+  genotypeConcordanceContingencyMetrics: Option[URI] = None,
+  genotypeConcordanceDetailMetrics: Option[URI] = None,
+  genotypeConcordanceSummaryMetrics: Option[URI] = None,
+  grnIdat: Option[URI] = None,
+  gtcFile: Option[URI] = None,
+  haplotypeDatabaseFile: Option[URI] = None,
+  hetHomvarRatio: Option[Double] = None,
+  hetPct: Option[Double] = None,
+  imagingDate: Option[OffsetDateTime] = None,
+  isLatest: Option[Boolean] = None,
+  isZcalled: Option[Boolean] = None,
+  modifiedAt: Option[OffsetDateTime] = None,
```

Comment:
These are timestamps in the database for when the row was created.  Don't we have this in clio?